rbc/tests/heavydb/test_imports.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+classes = [
+    'Array',
+    'ArrayPointer',
+    'Bytes',
+    'Column',
+]
+
+
+@pytest.mark.parametrize('cls', classes)
+def test_heavydb_imports(cls):
+    try:
+        exec(f'from rbc.heavydb import {cls}')
```

Comment:
I think `importlib` should be used instead.

Reply:
Done!